parquet-column/src/main/java/org/apache/parquet/internal/column/columnindex/ColumnIndexBuilder.java

```diff
@@ -298,24 +295,22 @@ public <T extends Comparable<T>> PrimitiveIterator.OfInt visit(NotEq<T> notEq) {
     public <T extends Comparable<T>> PrimitiveIterator.OfInt visit(In<T> in) {
       Set<T> values = in.getValues();
       IntSet matchingIndexesForNull = new IntOpenHashSet();  // for null
-      Iterator<T> it = values.iterator();
-      while(it.hasNext()) {
-        T value = it.next();
-        if (value == null) {
-          if (nullCounts == null) {
-            // Searching for nulls so if we don't have null related statistics we have to return all pages
-            return IndexIterator.all(getPageCount());
-          } else {
-            for (int i = 0; i < nullCounts.length; i++) {
-              if (nullCounts[i] > 0) {
-                matchingIndexesForNull.add(i);
+      for (T value : values) {
+          if (value == null) {
```

Comment:
Nit: Let's modify the indent spaces to 2 and ensure consistency?

Reply:
Ah, good point. I've updated the PR. Thanks for catching this 👍